great_expectations/data_context/templates.py

```diff
@@ -130,17 +130,5 @@ def dump(self, data, stream=None, **kw):  # type: ignore[explicit-override] # FI
 """
 )
 
-USAGE_STATISTICS_ENABLED = """
-analytics_enabled: True
-"""
-
-USAGE_STATISTICS_DISABLED = """
-analytics_enabled: False
-"""
```

Comment:
Why this removal?

Reply:
This field is what will show up in the config by default. The approach in this PR is for both the env var and the config variable to be optional bools, so this variable would not be present unless a user specifically sets it to True or False. Now the default value is handled directly in `_determine_analytics_enabled`.

It's been a few weeks since I looked at this, but I think there was also some additional complexity if we continued to set a default here in terms of implementation details. But the main thing is just philosophically treating this as None OR what a user explicitly selects.